### ISSUE TITLE
0.4.1 panic: runtime error: index out of range

### ISSUE THREAD

**Issue by seanmars** [isacikgoz/gitbatch#82]
```sh
go version
go version go1.11.5 darwin/amd64

gitbatch --version
gitbatch version 0.4.1
```

```log
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/isacikgoz/gitbatch/gui.(*Gui).renderRemotes(0xc42016ee60, 0xc42023c270, 0x0, 0x0)
	/home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/sideviews.go:98 +0x49c
github.com/isacikgoz/gitbatch/gui.(*Gui).renderSideViews(0xc42016ee60, 0xc42023c270, 0xc420283ab8, 0x1)
	/home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/sideviews.go:56 +0xac
github.com/isacikgoz/gitbatch/gui.(*Gui).renderMain(0xc42016ee60, 0x0, 0x0)
	/home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/repositoriesview.go:29 +0x1ce
github.com/isacikgoz/gitbatch/gui.(*Gui).repositoryUpdated.func1(0xc4201d4000, 0x1, 0xc4200efb40)
	/home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/repositoriesview.go:35 +0x2a
github.com/jroimartin/gocui.(*Gui).MainLoop(0xc4201d4000, 0xc4201d4000, 0x0)
	/home/isacikgoz/.local/share/go/src/github.com/jroimartin/gocui/gui.go:377 +0x1e0
github.com/isacikgoz/gitbatch/gui.(*Gui).Run(0xc42016ee60, 0x0, 0x0)
	/home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/gui.go:140 +0x1ee
main.run(0x0, 0x0, 0x0, 0x170b753, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/isacikgoz/git/gitbatch/main.go:52 +0x122
main.main()
	/home/isacikgoz/git/gitbatch/main.go:24 +0xc2
```

**Comment by isacikgoz** [comment]
Thanks for the feedback @seanmars I guess you downloaded `gitbatch'  from the releases page? If so, this issue may be duplicate of #71. You can install with `go get -u github.com/isacikgoz/gitbatch` to newer version. Or in case you want to wait for the next release, I will be releasing the newer version in the near future.

**Comment by gregf** [comment]
I just installed with go get -u and i'm getting the same error.

**Comment by isacikgoz** [comment]
Thanks for the reporting @gregf. Is it @ version `0.4.2`?

**Comment by gregf** [comment]
@isacikgoz my mistake this was fixed in 0.4.2 i ran it on 0.4.1 by mistake.

**Comment by seanmars** [comment]
> Thanks for the feedback @seanmars I guess you downloaded `gitbatch` from the releases page? If so, this issue may be duplicate of #71. You can install with `go get -u github.com/isacikgoz/gitbatch` to newer version. Or in case you want to wait for the next release, I will be releasing the newer version in the near future.

Yes, i downloaded from the release page. It's because take too long time by install with `go get -u github.com/isacikgoz/gitbatch` and i don't know why, so i install it by the release page.

But it's good to know that will be fixed on next version.
Well done.
